lingua_franca/lang/parse_common.py

```diff
@@ -385,3 +386,20 @@ def replace_right(source, target, replacement, replacements=None):
             extract = extract_handler(to_parse, short_scale, ordinals)
     numbers.reverse()
     return numbers
+
+
+class DurationResolution(Enum):
```

Comment:
Could this move "up" to common, as TimeResolution, maybe add the "delta" members to an override in here?

We're going to want the same enum in the date and duration *formatters* down the road.

```
class TimeResolution(Enum):
    MICROSECONDS = 0
    SECONDS = 1
    MINUTES = 2
    ...